features/templates.feature

```diff
@@ -0,0 +1,19 @@
+@templates
+Feature: Templates
+
+    Description from screwdriver-cd/screwdriver#470
+
+    Background:
+        Given "calvin" has admin permission to the template
+
+    @ignore
+    Scenario Outline: A template owner wants to validate their template
```

Comment:
hmm, from this PR: https://github.com/screwdriver-cd/screwdriver/pull/480/files, looks like we don't enforce any permission control for call validate endpoint. Looks like we don't need `Backgournd: Given "calvin" has admin permission to the template` for these scenarios?